logging/logrus/middleware_test.go

```diff
@@ -36,6 +36,16 @@ func TestLogrusMiddlewareSuite(t *testing.T) {
 			http_logrus.WithDecider(func(w httpwares.WrappedResponseWriter, r *http.Request) bool {
 				return r.URL.Path != "/blah"
 			}),
+			http_logrus.WithRequestFieldExtractor(func(req *http.Request) map[string]interface{} {
+				return map[string]interface{}{
+					"http.request.custom": "test",
```

Comment:
Can you make this function extract some information from the passed request instead of using "test"?